### PR TITLE
Telomerase activities predicion methods

### PR DESCRIPTION
**Pull request by NNoureen**
### The purpose here is to add methodology section for Issue # 76 (Methods: telomerase activity prediction)

_Briefly describe your changes here._

We have added methodology for quantification of telomerase activities in 03.methods.md file

### Issue

_What GitHub issue does your pull request address?_
Issue # 76 (Methods: telomerase activity prediction)

#### Which areas should receive a particularly close look?
Methods for quantification of telomerase activities

#### Is there anything that you want to discuss further?

NO

#### Is the pull request ready for review?
YES

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [ YES] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [YES ] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [YES ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [YES ] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [ YES] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### CODE REVIEW

File: build/assets/custom-dictionary.txt [hunk 1]
```diff
@@ -243,6 +243,8 @@ translocations
 trinucleotide
 tsv
 xCell
+telomerase
+Poly-A
```
Comment (jashapiro) [inline build/assets/custom-dictionary.txt:247]: Removing this dictionary element and changing (in `content/03.methods.md`) to "poly-A", consistent with the rest of the document.
```suggestion
```
It turns out this was also causing the build error, as spelling words can't contain a `-`

File: content/03.methods.md [hunk 2]
```diff
@@ -385,3 +385,9 @@ Resulting calls were added to the clinical metadata as `germline_sex_estimate`.
 Certain analyses required that we select only a single representative specimen for each individual.
 In these cases, we prioritized primary tumors and those with whole-genome sequencing available.
 If this filtering still resulted in multiple specimens, we selected from the remaining set randomly.
+
+
+
+### Quantify Telomerase Activity using Gene Expression Data
+
+We predicted telomerase activity of pediatric brain tumor samples using our recently developed method [EXTEND] (https://github.com/NNoureen/EXTEND). In brief, EXTEND estimates telomerase activity by scoring a 13-gene signature. We validated EXTEND using cell line datasets and demonstrated its superior performance to TERT expression in predicting telomerase activity (manuscript under preparation). Here we tested the concordance of EXTEND scores between counts and FPKM gene expression values, as well as Poly-A and rRNA  depletion protocols. Telomerase activity scores were compared across different brain tumor histological subtypes, with a minimum of 2 cases per group. Telomerase activities have also been compared for different Medulloblastoma molecular subtypes.
```
Comment (jashapiro) [inline content/03.methods.md:393]: Split to one sentence per line
```suggestion
We predicted telomerase activity of pediatric brain tumor samples using our recently developed method [EXTEND] (https://github.com/NNoureen/EXTEND).
In brief, EXTEND estimates telomerase activity by scoring a 13-gene signature.
We validated EXTEND using cell line datasets and demonstrated its superior performance to TERT expression in predicting telomerase activity (manuscript under preparation).
Here we tested the concordance of EXTEND scores between counts and FPKM gene expression values, as well as poly-A and rRNA  depletion protocols.
Telomerase activity scores were compared across different brain tumor histological subtypes, with a minimum of 2 cases per group.
Telomerase activities have also been compared for different Medulloblastoma molecular subtypes.
```

**Comment by sjspielman** [comment]
Hi @NNoureen, thanks for this addition! It's a great start, but a lot more detail is needed to fully explain the approach used. I'll take each sentence at a time below:

> We predicted telomerase activity of pediatric brain tumor samples using our recently developed method [EXTEND] (https://github.com/NNoureen/EXTEND). In brief, EXTEND estimates telomerase activity by scoring a 13-gene signature. 

This needs substantially more detail - how is scoring performed? What is a 13-gene signature? What specific type of data does EXTEND use (RNAseq, for example?). Especially for an unpublished, this information is needed to fully understand the results EXTEND produces. The goal is not to entirely recapitulate what will be explained in the EXTEND manuscript, but there should be enough detail for readers to have a sense of how it works. This should be at least 2-3 more sentences.

> We validated EXTEND using cell line datasets and demonstrated its superior performance to TERT expression in predicting telomerase activity (manuscript under preparation). 

This sentence strikes me as not necessarily belonging to *this* manuscript (it would belong to the EXTEND MS itself). It is reasonable to say here that EXTEND has been shown to outperform TERT. If you want to describe the EXTEND benchmarking at all, you'll need to cite and explain, *briefly*, what TERT is.

> Here we tested the concordance of EXTEND scores between counts and FPKM gene expression values, as well as Poly-A and rRNA depletion protocols. 

This needs to be much more specific - what is meant by "tested"? What type of statistical test was performed? I'm also a little unclear based on this wording on which specific comparisons were made. If you could try to rephrase this to be a but more explicit there? Further, it should be explained where the data used is from - Is it all samples? It is all individuals/participants? 

> Telomerase activity scores were compared across different brain tumor histological subtypes, with a minimum of 2 cases per group. Telomerase activities have also been compared for different Medulloblastoma molecular subtypes.

This needs more detail - how was comparison performed? In this case, you did a series of two-sample t-tests with a Bonferroni correction for multiple testing, but only for Medulloblastoma. For the more general subtype comparison, there was no statistical testing performed (as of yet?, but one could conceivably run an ANOVA here or similar), only visualization.

File: build/assets/custom-dictionary.txt (hunk 1 above)
Comment (NNoureen) [reply build/assets/custom-dictionary.txt:247]: Sure, I will do the changes as suggested. Thanks.

**Comment by NNoureen** [comment]
Thanks for your suggestions. I will add all the necessary details.

**Comment by jaclyn-taroni** [comment]
This pull request addresses #76

**Comment by sjspielman** [comment]
Hi @NNoureen, 
Thanks for this update! The details are much improved at this point. A couple tiny updates:
+ Rather than using the phrase "our preprint", please simply say "in reference" and cite the paper as normal. 
+ You will need to capitalize "Bonferroni" and "Spearman" 
+ I recommend rephrasing "using t-test" to "using a series of pairwise two-sample t-tests." In addition, the manubot platform is raising some spelling errors. You'll want to add these words to the [custom dictionary](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/773b9206b691433a712a9dd57dcde51aa77d658b/build/assets/custom-dictionary.txt) file.

**Comment by NNoureen** [comment]
Hi Dr. Stephanie,

Thanks for your comments. I will update all the details mentioned.

Thanks,
Nighat
